File: HelloWorldAndroidJava/app/src/main/java/maia/orlewilson/helloworldandroidjava/MainActivity.java
package maia.orlewilson.helloworldandroidjava;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    private Button btnLogin;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_layout);

        btnLogin = (Button) findViewById(R.id.btnLogin);

        btnLogin.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                EditText edUser = findViewById(R.id.loginUser);
                EditText edPassword = findViewById(R.id.loginPassword);

                if (edUser.getText().toString().equals("Orlewilson") && 
                        edPassword.getText().toString().equals("12345")){
                    showMessage("Bem-vindo, login realizado com sucesso.");
                } else {
                    showMessage("Login e senha incorretos.");
                }
            }
        });

    }

    private void showMessage(String msg){
        Toast.makeText(MainActivity.this, msg, Toast.LENGTH_SHORT).show();
    }
}

File: HelloWorldAndroidKotlin/app/src/main/java/maia/orlewilson/helloworldandroidkotlin/MainActivity.kt
package maia.orlewilson.helloworldandroidkotlin

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.os.Message
import android.view.View
import android.widget.Button
import android.widget.EditText
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val btLogin = findViewById<Button>(R.id.btnLogin)

        btLogin.setOnClickListener {
            val edLogin = findViewById<EditText>(R.id.loginUser)
            val edPassword = findViewById<EditText>(R.id.loginPassword)

            if (edLogin.text.toString().equals("Orlewilson") &&
                    edPassword.text.toString().equals("12345")){
                showMessage("Bem-vindo, login realizado com sucesso.")
            } else {
                showMessage("Login e senha incorretos.")
            }
        }
    }

    private fun showMessage(msg: String) {
        Toast.makeText(this, msg, Toast.LENGTH_SHORT).show()
    }
}
